simpleheat.js

```diff
@@ -53,7 +53,7 @@ simpleheat.prototype = {
     },
 
     radius: function (r, blur) {
-        blur = blur || 15;
+        blur = (blur == null ? 15 : blur)
```

Comment:
- `blur === undefined` would be a more precise comparison
- no need for parentheses
- semicolon at the end of line